docs/moduleAna5Utils.rst

```diff
@@ -0,0 +1,105 @@
+produce simulated **range-time diagrams** of the flow variables with respect to a radar field
+of view. With this, simulation results can be directly compared to radar measurements in terms
+of front position and inferred approach velocity. The colorcoding of the simulated
+**range-time** diagrams refers to average values of the chosen flow parameter
+(e.g. flow depth, flow velocity) at specified range gates. This colorcoding is not directly
+comparable to the intensity given in the range-time diagram from radar measurements.
+
+.. Note::
+  The data processing for the **tt-diagram** and the **range-time diagram** can be done
+  *during run time* of :py:mod:`com1DFA`, or as a postprocessing step. However, the second option
+  requires saving all the required time steps of the flow variable fields, which is much more
+  computationally expensive compared to the first option.
+
+To run
+~~~~~~~
+
+During run-time of :py:mod:`com1DFA`:
+
+* in your local copy of ``com1DFA/com1DFACfg.ini`` in [VISUALISATION] set `createRangeTimeDiagram`
+  to True and chose if you want a *TTdiagram* by setting this flag to True or in the case of a
+  simulated range-time diagram to False
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+* run  :py:mod:`runScripts.runThalwegTimeDiagram.py` or :py:mod:`runScripts.runRangeTimeDiagram.py`
+  and set the `preProcessedData` flag to `True`
+
+As a postprocessing step:
+
+* have a look at :py:mod:`runScripts.runThalwegTimeDiagram.py` and :py:mod:`runScripts.runRangeTimeDiagram.py`
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+The resulting figures can be found in ``avalancheDirectory/Outputs/ana5Utils``.
+
+
+.. figure:: _static/thalwegTime_FD.png
+    :width: 90%
+
+    Thalweg-time diagram example.
+
+
+.. Note::
+  The **tt-diagram** requires info on an avalanche path (see :ref:`moduleAna3AIMEC:ana3AIMEC: Aimec`).
+  The simulated **range-time diagram** requires info on the x, y coordinate of the radar location, a point
+  in the direction of the field of view, the aperture angle and the width of the range gates.
+  The maximum approach velocity indicated in the diagrams is computed as the ratio of the
+  distance travelled by the front and the respective time needed for two increments along the path/
+  radar's field of view.
+
+
+Theory
+~~~~~~~~~
+
+Thalweg-time diagram
+~~~~~~~~~~~~~~~~~~~~~~
+
+First, the flow variable result field is transformed into a path-following coordinate system.
+For this step, functions from :py:mod:`ana3AIMEC` are used.
+The distance to the *start of runout area point* is determined using a user defined threshold
+of the flow variable, giving the avalanche front position. The front positions defined with this
```

Comment:
I dont understand this sentence: *start of runout area point* is the beta point, right? What has it to do with the threshold to determine the avalanche (not only the front)?

Reply:
so it can be for flow depth 1m  or 5cm or just >0 so that it everywhere where there is avalanche flow